R/Quandldatabase.R

```diff
@@ -51,13 +51,7 @@ Quandl.database.bulk_download_url <- function(database_code, ...) {
 #' }
 #' @export
 Quandl.database.bulk_download_to_file <- function(database_code, filename, ...) {
-  if (dir.exists(filename)) {
```

Comment:
yeah sorry, @RaymondMcT wants to remove `dir.exists` because it's incompatible with the old release of R (2.7.x). He is okay with `httr` handling the errors